_posts/2020-09-19-github-actions-with-faasd.md

```diff
@@ -0,0 +1,218 @@
+---
+title: "Github Actions for OpenFaas functions with faasd"
```

Comment:
Thinking about the audience for this. Is anything specific to faasd, or can this be about OpenFaaS with brief notes where minor things diverge?

Reply:
I want it to be specific to faasd, maybe adding an update showcasing the logs, with how the function gets rebuilt, by adding a log statement to the handler?

Reply:
Given that this blog post isn't really about faasd, and that it is using normal OpenFaaS APIs, I would prefer to see it made more generic: `Deploying to OpenFaaS with GitHub Actions` for instance